media-libs/chromaprint/chromaprint-1.4.3.recipe

```diff
@@ -19,10 +19,13 @@ libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
 PROVIDES="
 	chromaprint$secondaryArchSuffix = $portVersion
 	lib:libchromaprint$secondaryArchSuffix = $libVersionCompat
+	cmd:fpcalc
```

Comment:
Shouldn't it read cmd:fpcalc$secondaryArchSuffix?

Reply:
Thanks, changed to `cmd:fpcalc$secondaryArchSuffix = $portVersion`